### ISSUE TITLE
problems with low accuracy

### ISSUE THREAD

**Issue by nassim-fox** [andreybicalho/ExtendedMorphologicalProfiles#2]
I'm getting a really low accuracy using 10% training data on Indian pines ( getting 31% OA ) although the paper says they're getting 93% OA.
even when using the default values you are using, i'm getting 40% OA.

**Comment by andreybicalho** [comment]
Have you reproduced the exact same code with the exact same data?

**Comment by andreybicalho** [comment]
> I'm getting a really low accuracy using 10% training data on Indian pines ( getting 31% OA ) although the paper says they're getting 93% OA.
> even when using the default values you are using, i'm getting 40% OA.

Have you reproduced the exact same code with the exact same data?

**Comment by nassim-fox** [comment]
Of course I executed the same code and the exacte same data on google Colab.

**Comment by Tifou-Kerdjou** [comment]
I had the same problem, I thought it was an error in the code I used

**Comment by andreybicalho** [comment]
My suggestion is to try to run it in another environment other than Google Colab... I've just run it again, fixed some minor import, and added the dependency requirements so you use the exact same dependency I'm using right now. I hope it helps.
